zarr/tests/util.py

```diff
@@ -36,3 +39,10 @@ def __setitem__(self, key, value):
     def __delitem__(self, key):
         self.counter['__delitem__', key] += 1
         del self.wrapped[key]
+
+
+def skip_test_env_var(name):
+    """ Checks for environment variables indicating whether tests requiring services should be run
+    """
+    value = os.environ.get(name, '0')
```

Comment:
Should we handle empty strings? If so, this might help.

```suggestion
    value = os.environ.get(name, '') or '0'
```

Reply:
Hmm I'm not sure. Is an empty string a common pattern for disabling things with an env var? If so, then it's probably worth supporting here. No strong opinion from me one way or the other

Reply:
It's more so that it is not a common pattern for enabling things. So the concern would be someone accidentally enabling these.

That said, let's leave it since it was already this way and we can follow-up in another issue/PR.